INSTALLING.md

```diff
@@ -218,16 +218,10 @@ you need to follow these instructions:
 First build and install `pmemkv` as described above. Then, install client libraries for Memcached:
 
 ```sh
-cd ~
-mkdir work
-cd work
-wget https://launchpad.net/libmemcached/1.0/0.21/+download/libmemcached-0.21.tar.gz
-tar -xvf libmemcached-0.21.tar.gz
-mv libmemcached-0.21 libmemcached
-cd libmemcached
-./configure
-make
-su -c 'make install'
+# on Fedora:
+sudo dnf install memcached
+# or on Ubuntu:
+sudo apt install memcached
```

Comment:
Are you sure this is enough? I think you need libmemcached-dev too.

I think this PR should include docker and run-build.sh updates.

Reply:
I even checked the deps, but forgot to update them here.
Done.